### PR TITLE
Mirrorlist test changes for Issue #146

### PR DESCRIPTION
**Pull request by AlanMarshall**
This test originally attempted use the dvd-iso to demonstrate installation of a graphical server from the mirrorlist.  However, after mirrorlist is selected as the Installation Source the option to install "Server with GUI" (graphical-server)  disappears from the Base Environment list and only "server", "minimal" and "custom" package sets remain. This is the subject of Issue #146.

The issue is resolved for 9.1 by selecting "server" from the Base Environment list but as mentioned in #146 there is a remaining issue with resolving the mirrorlist URL for 8.7. This is potentially a problem with a returned mirror, not the test code. New attempts to download repository metadata, presumably from a different returned mirror, were eventually successful.

In conclusion, this is a fix for 9.1 but not 8.7 and Issue #146 needs to be pursued further.

This was tested using:
```
openqa-cli api -X POST isos ISO=Rocky-9.1-20221214.1-x86_64-dvd.iso ARCH=x86_64 DISTRI=rocky FLAVOR=universal VERSION=9.1 BUILD=-"$(date +%Y%m%d.%H%M%S).0"-9.1-20221214.1-universal TEST=install_mirrorlist_graphical
openqa-cli api -X POST isos ISO=Rocky-8.7-x86_64-dvd1.iso ARCH=x86_64 DISTRI=rocky FLAVOR=universal VERSION=8.7 BUILD=-"$(date +%Y%m%d.%H%M%S).0"-8.7-20221110-universal TEST=install_mirrorlist_graphical
```
Both tests should pass but sporadic failures of 8.7 at the "_check_install_source" stage are expected.

### CODE REVIEW

**Review by tcooper — APPROVED** [review]
Testing with...

```bash
$ openqa-clone-custom-git-refspec https://github.com/rocky-linux/os-autoinst-distri-rocky/pull/157 https://openqa.rockylinux.org/tests/11667
Created job #11805: rocky-9.1-universal-x86_64-Build20230313-Rocky-9.1-x86_64.0-install_mirrorlist_graphical@64bit -> https://openqa.rockylinux.org/t11805
```

```bash
$ openqa-cli api jobs/11805 -p
```

```json
{
   "job" : {
      "assets" : {
         "iso" : [
            "Rocky-9.1-20221214.1-x86_64-dvd.iso"
         ]
      },
      "assigned_worker_id" : 3,
      "blocked_by_id" : null,
      "children" : {
         "Chained" : [],
         "Directly chained" : [],
         "Parallel" : []
      },
      "clone_id" : null,
      "group_id" : null,
      "has_parents" : 0,
      "id" : 11805,
      "name" : "rocky-9.1-universal-x86_64-BuildAlanMarshall_os-autoinst-distri-rocky_157-install_mirrorlist_graphical@AlanMarshall_os-autoinst-distri-rocky_mirrorlist@64bit",
      "parents" : {
         "Chained" : [],
         "Directly chained" : [],
         "Parallel" : []
      },
      "parents_ok" : 1,
      "priority" : 50,
      "result" : "none",
      "settings" : {
         "ARCH" : "x86_64",
         "ARCH_BASE_MACHINE" : "64bit",
         "BACKEND" : "qemu",
         "BUILD" : "AlanMarshall\/os-autoinst-distri-rocky#157",
         "CASEDIR" : "https:\/\/github.com\/AlanMarshall\/os-autoinst-distri-rocky.git#mirrorlist",
         "CLONED_FROM" : "https:\/\/openqa.rockylinux.org\/tests\/11667",
         "CURRREL" : "9",
         "DISTRI" : "rocky",
         "FLAVOR" : "universal",
         "HDDSIZEGB" : "15",
         "ISO" : "Rocky-9.1-20221214.1-x86_64-dvd.iso",
         "LOCATION" : "https:\/\/dl.rockylinux.org\/pub\/rocky\/9.1",
         "MACHINE" : "64bit",
         "MIRRORLIST_GRAPHICAL" : "1",
         "NAME" : "00011805-rocky-9.1-universal-x86_64-BuildAlanMarshall_os-autoinst-distri-rocky_157-install_mirrorlist_graphical@AlanMarshall_os-autoinst-distri-rocky_mirrorlist@64bit",
         "NEEDLES_DIR" : "rocky\/needles",
         "NICTYPE_USER_OPTIONS" : "net=172.16.2.0\/24",
         "PART_TABLE_TYPE" : "mbr",
         "PRODUCTDIR" : "os-autoinst-distri-rocky",
         "QEMUCPU" : "Nehalem",
         "QEMUCPUS" : "2",
         "QEMURAM" : "2048",
         "QEMU_HOST_IP" : "172.16.2.2",
         "QEMU_VIDEO_DEVICE" : "virtio-vga",
         "QEMU_VIRTIO_RNG" : "1",
         "TEST" : "install_mirrorlist_graphical@AlanMarshall\/os-autoinst-distri-rocky#mirrorlist",
         "TEST_SUITE_NAME" : "install_mirrorlist_graphical",
         "TEST_TARGET" : "ISO",
         "VERSION" : "9.1",
         "WORKER_CLASS" : "qemu_x86_64",
         "XRES" : "1024",
         "YRES" : "768"
      },
      "state" : "running",
      "t_finished" : null,
      "t_started" : "2023-03-13T16:54:10",
      "test" : "install_mirrorlist_graphical@AlanMarshall\/os-autoinst-distri-rocky#mirrorlist"
   }
}
```

In above job JSON the `PACKAGE_SET` is not set to `server`. This indicates that `openqa-clone-custom-git-refspec` method will not work PRs that make changes in whole or in part of `templates.fif.json` and that such tests require local modification of `templates.fif.json` along with `fifloader.py` or must be run outside of the production openQA instance.

The desired behavior would be achieved by `POST`ing with the `PACKAGE_SET` variable or by adding that variable in an `openqa-clone-job` command (which has already been done).

Based on above I am approving this PR.

**Review by akatch — APPROVED** [review]
The test for 8.7 fails as indicated but 9.1 passes. Approved!
